### ISSUE TITLE
cannot get `linuxFxVersion` property for WebApp

### ISSUE THREAD

**Issue by juniwang** [Azure/azure-sdk-for-java#1761]
After I get a web app though `WebApp webApp = azureClient.webApps().getByResourceGroup(resourceGroup, appServiceName);`, I cannot find a way to get the property `linuxFxVersion` which indicates which docker image was used.  

But in debug model, I CAN see that there is a field `webapp.siteConfig.linuxFxVersion` which means the instance do has the property but no public getter to access it.  

I tried `((WebAppImpl) webApp).siteConfig.linuxFxVersion()` which works in the debug window only. Because `WebAppImpl` is not public accessable so I cannot import it into my codes.

So how can I get this field for now? Do you have any plan to make it  public to better support app service on Linux?

**Comment by jianghaolu** [comment]
Hi @juniwang this is a new property. Will expose in the next minor release. 

For now, please access the `siteConfig` property through reflection.

**Comment by anuchandy** [comment]
@jianghaolu this is also accessible via inner right? like
```java
WebApp webApp = ...
String version = webApp.inner().siteConfig().linuxFxVersion();
```

**Comment by jianghaolu** [comment]
@anuchandy Unfortunately it's not. The `siteConfig()` on `inner()` is not populated from a GET response.

**Comment by anuchandy** [comment]
ah ok got it.

**Comment by juniwang** [comment]
@jianghaolu thanks. Will reflection be a robust way? I'm worrying about that private fields might be obfuscated after compile.

**Comment by jianghaolu** [comment]
@juniwang Could be. 

There's another way. You can call this method as a workaround: 
```java
azureClient.webApps().inner().getConfiguration(app.resourceGroupName(), app.name());
```

**Comment by martinsawicki** [comment]
released in v1.3
